### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by Ishodnikov**

### CODE REVIEW

File: src/Message/CompletePurchaseResponse.php [hunk 1]
```diff
@@ -50,9 +50,8 @@ public function generateSignature()
     public function getCustomFields()
     {
         $fields = array_filter([
-            'Shp_TransactionId' => $this->getTransactionId(),
-            'Shp_Client' => $this->getClient(),
```
Comment (SilverFire) [inline src/Message/CompletePurchaseResponse.php:54]: Why did you remove `Shp_Client`?

**Comment by SilverFire** [comment]
Thank you for taking care of this package!

**Comment by hiqsol** [comment]
Thank you!
I've merged your changes in 3.0 branch.
It can be installed with:

```json
    "require": {
        "hiqdev/omnipay-robokassa": "^3.0@dev"
    },
```

**Comment by SilverFire** [comment]
https://docs.robokassa.ru/en#2540

According to the documentation, the signature must include `Shp_login` field, if it was passed.

File: src/Message/CompletePurchaseResponse.php (hunk 1 above)
Comment (Ishodnikov) [reply src/Message/CompletePurchaseResponse.php:54]: What is Shp_Client for?

Comment (SilverFire) [reply src/Message/CompletePurchaseResponse.php:54]: Any `Shp_*`is supported by RoboKassa as a custom field.
We use this field in our implementation of integration, but it's optional. It's OK if you don't fill it

**Comment by Ishodnikov** [comment]
> https://docs.robokassa.ru/en#2540
> 
> According to the documentation, the signature must include `Shp_login` field, if it was passed.

Let's add it
